app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Post;
+
+class PostController extends Controller
+{
+    public function store(Request $request)
+    {
+
+      $validatedData = $request->validate([
+          'name' => 'required',
+          'description' => 'required'
+      ]);
+
+      $user = auth()->user();
```

Comment:
Grab the user from the request instead `$request->user()`

Reply:
Agree, we can actually do this: `$post->user_id = $request->user()->id;` to minimize variable declaration.